scripts/purge-sqs.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -euox pipefail
```

Comment:
Be careful about `set -x` here. I think the secrets will get exposed to stdout because the variables are interpolated before the `export` is called -- but I could be wrong.

Reply:
Yep:
```console
$ cat foo.sh
#!/bin/bash
set -x
export FOO=${BAR}
$ BAR=baz bash foo.sh
+ export FOO=baz
+ FOO=baz
$
```

We don't particularly want secrets getting echoed to standard output, especially if they're going via Drone.

Reply:
Removed -x from all scripts.